NSS-Day-Cohort-66/rare-client-dapper-pirates-1
# category list

I created a category directory in which I defined a `Category` component. It displays a list of categories in alphabetical order with an edit and delete button next to each category *(this is admin view, only admins will have the edit and delete functionality)*. Set up routing within the `ApplicationView` file so that when the user is at the browser url "/category-manager" the `Category` component displays in the browser.

*THE CATEGORIES ARE NOT CLICKABLE YET AND THE EDIT AND DELETE BUTTONS DO NOT HAVE FUNCTIONALITY, THOSE ARE HANDLED BY OTHER TICKETS*

## Steps to test
1. Pull down the `he-posts-by-category` branch and switch to it
2. If tailwind is not already installed run this command in your terminal ` npm install -D tailwindcss postcss autoprefixer`
3. Ensure that your server side code debugger is running with the most recent version of the `category.py` view, this means that a `list()` method and a `retrieve()` method will be defined and functional
4. Run `npm start` to deploy the website or if the website is already running on your browser, refresh the page 
5. Open your chrome dev tools and keep them open while testing functionality to check for any errors that may come up in the console log, if you do see errors in the console log refresh the browser to ensure that all errors you see are current (there shouldn't be any errors, so if there are after refreshing do not approve this pull request)
6. Click the 'Category Manager' tab in the navigation bar on the browser
7. Verify that you see a list of categories in alphabetical order, next to each category will be a gear and a trash icon, those are the edit and delete buttons **they are NOT functional**
8. Ensure there are no errors that came up in the console log before approving pull request
